tests/unit/workflows/test_workflows_tasks_submission.py

```diff
@@ -35,9 +41,51 @@ class DummyEng(object):
     pass
 
 
+def fake_is_arxiv_paper(obj):
+    return True
+
+
 def test_add_note_entry_does_not_add_value_that_is_already_present():
     obj = StubObj({'public_notes': [{'value': '*Temporary entry*'}]}, {'core': 'something'})
     eng = DummyEng()
 
     assert add_note_entry(obj, eng) is None
     assert {'public_notes': [{'value': '*Temporary entry*'}]} == obj.data
+
+
+@mock.patch('inspirehep.modules.workflows.tasks.submission.is_arxiv_paper',
+    side_effect=fake_is_arxiv_paper)
```

Comment:
BTW this is an example of Classicist vs Mockist testing: http://martinfowler.com/articles/mocksArentStubs.html#ClassicalAndMockistTesting.